go.mod

```diff
@@ -1,8 +1,9 @@
-module github.com/vishvananda/netlink
+module github.com/xaionaro-go/netlink
```

Comment:
?

Reply:
@aboch : Yes, sorry. My PR was closed (without merging) so I forked the project. I'll cleanup that things in few minutes.

Reply:
@aboch : I've just removed the garbage.

Reply:
I'll also add an unit-test, few minutes.

Reply:
Added an unit-test:

```
$ sudo go test ./... -count=1 -run=TestLinkAddDelWireguard
ok      github.com/vishvananda/netlink  0.045s
ok      github.com/vishvananda/netlink/nl       0.002s [no tests to run]
```